src/apify/_configuration.py

```diff
@@ -262,12 +263,13 @@ class Configuration(CrawleeConfiguration):
     ] = 4321
 
     standby_url: Annotated[
-        str,
+        str | None,
+        BeforeValidator(validate_http_url),
         Field(
             alias='actor_standby_url',
             description='URL for accessing web servers of Actor runs in Standby mode',
         ),
-    ]
+    ] = None
```

Comment:
It might be worth it to use some nonsense URL here instead, just for the ease of use.